### PR TITLE
Fix/projection report

### PR DESCRIPTION
**Pull request by seb231**

### CODE REVIEW

File: src/projection_report.R [hunk 1]
```diff
@@ -72,15 +95,23 @@ loadfonts(device = 'pdf')
...
-          geom_line(data = actuals, aes(x = date, y = value)) +
+          geom_line(data = actuals, aes(x = date, y = value, colour = variable)) +
           geom_line(data = projected, aes(x = date, y = median), linetype = 2) +
           geom_ribbon(data = projected, aes(x = date, ymin = lower.ci, ymax = upper.ci), fill = "gray", alpha = 0.3) +
           geom_ribbon(data = projected, aes(x = date, ymin = q1, ymax = q3), fill = "gray", alpha = 0.3) +
           geom_vline(xintercept = train_from, color = "red", linetype = 2) +
           theme_mastodon +
-          scale_color_manual(values = colours) +
-          labs(title = "CiC", x = "Date", y = "CiC"))
+          #scale_color_manual(values = colours) +
```
Comment (henrygarner) [inline src/projection_report.R]: Just checking: are the default colours really the ones we want?

File: src/projection_report.R [hunk 2]
```diff
@@ -173,16 +206,23 @@ loadfonts(device = 'pdf')
...
-            geom_line(data = actuals, aes(x = date, y = value)) +
+            geom_line(data = actuals, aes(x = date, y = value, colour = variable)) +
             geom_line(data = projected, aes(x = date, y = median), linetype = 2) +
             geom_ribbon(data = projected, aes(x = date, ymin = lower.ci, ymax = upper.ci), fill = "gray", alpha = 0.3) +
             geom_ribbon(data = projected, aes(x = date, ymin = q1, ymax = q3), fill = "gray", alpha = 0.3) +
-
             geom_vline(xintercept = train_from, color = "red", linetype = 2) +
             theme_mastodon +
-            scale_color_manual(values = colours) +
+            #scale_color_manual(values = colours) +
```
Comment (henrygarner) [inline src/projection_report.R]: Just checking: are the default colours really the ones we want?

**Review by henrygarner — APPROVED** [review]
LGTM, one query 🚢

File: src/projection_report.R (hunk 2 above)
Comment (seb231) [reply src/projection_report.R]: Ah right, thanks for flagging this @henrygarner and I don’t want to waste too much time on it, but to be able to plot the two “real” datasets I had to combine them (as you’ve seen) and create a column which differentiates between them. It was a problem between a discrete and a continuous datasets. Colouring only then worked by declared the column variable as where colours are dictated. To get this to work I had to comment out the scale_color_manual . So happy to use the non-default colours but I can’t figure out how to apply this

**Pull request merged**